forui/test/src/widgets/badge_test.dart

```diff
@@ -0,0 +1,71 @@
+@Tags(['golden'])
+library;
+
+import 'package:flutter/material.dart';
```

Comment:
Do we need material here?

Reply:
Yes since `Colors.blueAccent` is from Material. In general, I think it's fine to import Material in tests.